### PR TITLE
Add Pagination

### PR DESCRIPTION
**Pull request by nanu-c**
Add pagination support for data-tables. @JoelForamitti this doesn't include the filter because we have the filter disabled for most of the fields.

### CODE REVIEW

**Comment by JoelForamitti** [comment]
> Add pagination support for data-tables. @JoelForamitti this doesn't include the filter because we have the filter disabled for most of the fields.

@nanu-c can you try with this branch: https://github.com/MILA-Wien/collectivo/pull/76

**Review by JoelForamitti — CHANGES_REQUESTED** [review]
Some filters send the wrong request path with icontains instead of the actually chosen filter. E.g.: The full request path looks e.g. like this: `/api/members/tags/?offset=0&limit=50&search=&id__icontains=true`

I found this problem for the following three filters:
- equals should be `=`
- in list should be `__in=` 
- is empty should be `__isnull=`

**Comment by JoelForamitti** [comment]
The e2e test fails because there are not enough members to test pagination. Please test tags instead, where the latest version of https://github.com/MILA-Wien/collectivo/pull/76 comes with over 100 test tags.

**Comment by nanu-c** [comment]
> Some filters send the wrong request path with icontains instead of the actually chosen filter. E.g.: The full request path looks e.g. like this: `/api/members/tags/?offset=0&limit=50&search=&id__icontains=true`
> 
> I found this problem for the following three filters:
> 
>     * equals should be `id=`
> 
>     * in list should be `id__in=`
> 
>     * is empty should be `id__isnull=`

This is resolved now.

**Comment by nanu-c** [comment]
The problem here is, that even `/api/members/summary/?offset=0&limit=50&tags=1` doesn't work.

**Comment by nanu-c** [comment]
Okey filters are broken on all manyToMany fields, for example this `http://collectivo.local:8000/api/members/summary/?offset=0&limit=50&groups_interested__icontains=1`

**Comment by JoelForamitti** [comment]
> Okey filters are broken on all manyToMany fields, for example this `http://collectivo.local:8000/api/members/summary/?offset=0&limit=50&groups_interested__icontains=1`

Can you check if it works with this new branch? https://github.com/MILA-Wien/collectivo/pull/82

However, note that in your example there is still a syntax mistake, it should not be `groups_interested__icontains=1` but only `groups_interested=1`. Contains can only be used for strings, I think.

**Comment by nanu-c** [comment]
I have disabled the contains filter as it was also changed in the backend now -> https://github.com/MILA-Wien/collectivo/pull/82/files#diff-77542745d5ea7bb012c7c9f62c6452ab89169717c3f82b0e683a331d2eed4221R21

**Review by JoelForamitti — CHANGES_REQUESTED** [review]
Two more issues:

- The filter for single choice does not work any more (e.g. status in members). When I try to apply it, no request is send to the backend.
- The clear filters button does not work.

File: src/components/datatable/ObjectTable.vue [hunk 1]
```diff
@@ -264,7 +273,7 @@ function dataTableFilterModesToDjangoFilter(filterMode: string) {
                 :selectedItemsLabel="`${filterModel.value?.length} selected`"
                 :filter="true"
                 :placeholder="t('Select multiple choices')"
-                class="p-column-filter"
+                class="p-column-filter fitler-1"
```
[the remark this answers is not in the document]
Comment (nanu-c) [reply src/components/datatable/ObjectTable.vue]: I have removed this.

**Comment by nanu-c** [comment]
> Two more issues:
> 
>     * The filter for single choice does not work any more (e.g. status in members). When I try to apply it, no request is send to the backend.
-> solved
> 
>     * The clear filters button does not work.

The clear buttons have been removed, we need another solution for them. It can be done now by reloading the page

**Review by JoelForamitti — APPROVED** [review]

**Pull request merged**